src/express-middleware/image-proxy/file-cache.js

```diff
@@ -91,7 +91,8 @@ class FileCache {
     try {
       const dir = path.join(__dirname, `${key}`);
       const metadataPath = `${dir.replace(path.extname(dir), '.metadata')}`;
-      fs.outputFileSync(dir, data.data);
+      const imageData = data.data.replace(/^data:image\/\w+;base64,/, '');
+      fs.outputFileSync(dir, imageData, { encoding: 'base64' });
```

Comment:
Please don't write files encoded in base64. Please write them in the original binary format. No processing needed. If outputFileSync doesn't allow you to write binary files, find something else.